deployment/terraform/metrics.go

```diff
@@ -352,6 +353,29 @@ func (t *Terraform) setupMetrics(extAgent *ssh.ExtAgent) error {
 		return fmt.Errorf("error while uploading dashboard_json: output: %s, error: %w", out, err)
 	}
 
+	// Download dashboard v2 from and upload it
+	dashboardv2Resp, err := http.Get("https://grafana.com/api/dashboards/15582/revisions/latest/download")
+	if err != nil {
+		return fmt.Errorf("error downloading latest grafana v2 dashboard: %w", err)
+	}
+	defer dashboardv2Resp.Body.Close()
+
+	var dashboardV2Contents bytes.Buffer
+
+	_, err = io.Copy(&dashboardV2Contents, dashboardv2Resp.Body)
+	if err != nil {
+		return fmt.Errorf("error while reading dashboard v2: %w", err)
+	}
+
+	// Removes the DS_PROMETHEUS variable requirement to allow grafana to use the only prometheus
+	// datasource available in the load-test envionment.
+	re := regexp.MustCompile(`,\r?\n\s+\"uid\":\s?\"\$\{DS_PROMETHEUS\}\"`)
```

Comment:
Hmm .. I am not a big fan of this. Can we not use the `t.UploadDashboard` method to upload the dashboard, instead of manually copying over the json file to a folder?

Reply:
Oh yeah, I recall thinking about checking that method when I moved this upstream but I forget, sorry, let me make the changes.

Reply:
https://github.com/mattermost/mattermost-load-test-ng/pull/872/commits/f62d9970542f9f20ba8b256d11cd71172e19329a